configs/mot/deepsort/deepsort_faster-rcnn_fpn_4e_mot17-private-half.py

```diff
@@ -40,7 +40,8 @@
             num_samples=10,
             img_scale=(256, 128),
             img_norm_cfg=None,
-            match_score_thr=2.0),
```

Comment:
We already have a function for cosine similarity at [here](https://github.com/open-mmlab/mmtracking/blob/702c6ea603ea99364db035baadd3468b6004cf39/mmtrack/core/track/similarity.py#L5)